test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5222,42 +5263,74 @@ def save_configuration(self, outfile, mode='a'):
         """
         conf = {}
         sconf = {MGR_OBJ_SERVER: conf}
-
+        # save resourcedef file
         rd = os.path.join(self.pbs_conf['PBS_HOME'], 'server_priv',
                           'resourcedef')
-        self._save_config_file(conf, rd)
-
+        try:
```

Comment:
I didn't see code to load resourcedef file?

Reply:
I have removed resourcedef from both save & load config, because i am already using print server command to save resources & then loading them back using qmgr.